scripts/gz3d/README.md

```diff
@@ -0,0 +1,83 @@
+# GZ3D Dataset
+
+The [GZ3D](https://www.zooniverse.org/projects/klmasters/galaxy-zoo-3d) (Galaxy Zoo 3D) collected annotations of stars, bars, spiral arms, and the centers of 29,813 MaNGA galaxies. The respective PNG of the target is included in the dataset. It is published as a Value Added Catalog (VAC) as part of SDSS data release 17. Each segmentation is an aggregate of 15 human annotations. The associated catalog contains summaries of the respective classifications as collected by the Galaxy Zoo collaboration in relevant classification project. The original code on which [utils.py](./utils.py) is tracked [here]("https://github.com/CKrawczyk/GZ3D_production/"). See the [DESIGN.md](../../DESIGN.md) for the format description.
+
+## Data Download
+
+All data are downloaded from the [SDSS Science Archive Server (SAS)](https://data.sdss.org/sas/dr17/env/MANGA_MORPHOLOGY/galaxyzoo3d/v4_0_0/)
+
+[This script](./build_parent_sample.py) can be used to download GZ3D. To download a sample of 5, use the `--tiny` flag:
+```bash
+python build_parent_sample.py --tiny
+```
+This will download all of the FITS files, group them into folders for each healpix value they belong to. In each folder a single sample is then represented as a single hdf5 file. The full data is approximately 8GB in size. The direct download is likely rate limited by the host.
+
+> NOTE: The downloaded fits files are deleted if the `--keep_fits` flag is not set.
+
+<!-- We suggest downloading the data using globus from a registered host. -->
+
+<!-- ### sdss-access
+
+Alternatively, you can use the `access_tranfer` Python script to download the data with the [sdss-access](https://sdss-access.readthedocs.io/en/latest/) Python package, which uses parallelized `rsync` streams to download the data. This tool organizes the output download directory structure to mirror the official SDSS SAS.
+
+To download all summary files, cubes and maps, run:
+```bash
+python access_transfer.py --destination_path .
+``` -->
```

Comment:
This is legacy in case I decided to implement it. I tried and it turns out that getting sdss-access to work for this data was not worth it, given the current downloads work just fine (if a little bit slow). I think the solution is the globus from flatiron, but I don't know how to set that up / write about it.

tl;dr: `sdss-access` not actually used. 
```suggestion
```